torch/onnx/_internal/exporter/_torchlib/ops/core.py

```diff
@@ -45,3 +45,60 @@ def aten_add_complex(self: TReal, other: TReal, alpha: float = 1.0) -> TReal:
     """add.Tensor(Tensor self, Tensor other, *, Scalar alpha=1) -> Tensor"""
 
     return aten_add(self, other, alpha=alpha)
+
+
+@onnx_impl((aten.grid_sampler.default), trace_only=True)
```

Comment:
```suggestion
@onnx_impl(aten.grid_sampler.default, trace_only=True)
```

nit

Reply:
Do we need to align ops to torchlib (onnxscirpt)?

Reply:
Removed gridsample as an example